src/javascript/app/pages/cashier/payment_agent_withdraw.js

```diff
@@ -109,6 +110,9 @@ const PaymentAgentWithdraw = (() => {
                     // error handling
                     $agent_error.setVisibility(1);
                 }
+                if ($(field_ids.txt_amount).val()) {
+                    Validation.validate(form_id);
```

Comment:
don't we also need it for `$txt_agents.on('keyup'`?

Reply:
But it scrolls down to the `txtAmount` field every time a key is pressed

Reply:
then better use a different listener to value change instead of keyup on txt_agents if possible

Reply:
or, I like your idea better, let's go for that.

Reply:
Alright, done 👍